manage.py

```diff
@@ -1692,6 +1693,51 @@ def _add_rule_table_status_subparser(subparsers):
     rule_table_list_parser.add_argument('--debug', action='store_true', help=ARGPARSE_SUPPRESS)
 
 
+def _add_rule_table_stage_subparser(subparsers):
+    """Add the rule db stage subparser: manage.py rule-table stage"""
+    rule_table_stage_usage = 'manage.py rule-table stage'
+    rule_table_stage_desc = ("""
+StreamAlertCLI v{}
+Stage or unstage rules given their name as a space-separated list
+
+Command:
+
+    manage.py rule-table stage    Stage the rules provided in a space-separated list
+
+Optional Arguments:
+
+    --unstage                     Inverts the default action and unstages the rules provided
+    --debug                       Enable Debug logger output
+
+""".format(version))
+    rule_table_stage_parser = subparsers.add_parser(
+        'stage',
+        usage=rule_table_stage_usage,
+        description=rule_table_stage_desc,
+        formatter_class=RawTextHelpFormatter,
+        help=ARGPARSE_SUPPRESS)
+
+    rule_table_stage_parser.set_defaults(subcommand='stage')
+
+    rule_table_stage_parser.add_argument(
+        'rules',
+        action=UniqueSetAction,
+        default=set(),
+        help=ARGPARSE_SUPPRESS,
+        nargs='+'
+    )
+
+    rule_table_stage_parser.add_argument(
+        '-u', '--unstage',
```

Comment:
I am little hesitating to comment on this line. Because from users point, it is little confusing to unstage the rules after `stage` keyword
```
manage.py rule-table stage --unstage rule1 rule2
```

But if we change it to be a user friendly version, we will duplicate this subparser to support command like this
```
manage.py rule-table unstage rule1 rule2
```
What do you think?